### ISSUE TITLE
Incorrect expo-config(MODULE_NOT_FOUND) error

### ISSUE THREAD

**Issue by josmithua** [expo/vscode-expo#61]
### Description of the bug
The extension is reporting `expo-config(MODULE_NOT_FOUND)` for all plugins that I specify in `app.json`

### To Reproduce
0. `expo init`
1. Create `app.plugin.js` file with contents: `module.exports = (config) => config;`
2. In `app.json` add `expo.plugins` property: `    "plugins": ["./app.plugin"],`
3. Notice error:   
    ```
    Cannot find module '/Users/joshua/code/TestExpoApp/app.plugin.js' 
    expo-config(MODULE_NOT_FOUND)
    ```

#### Expected behavior
No error should appear

#### Actual behavior
Error appeared

### Additional context
```
Name: Expo Tools
Id: bycedric.vscode-expo
Description: Expo tools for vscode
Version: 0.7.3
Publisher: Cedric van Putten
VS Marketplace Link: https://marketplace.visualstudio.com/items?itemName=byCedric.vscode-expo
```

**Comment by guyzmo** [comment]
I reproduced that bug by launching a vscode here, and I had the same error. My local plugin works perfectly fine.
(though I cannot say I'm impacted, I'm a vim user 🤣 )

**Comment by tasugi** [comment]
I faced the same issue. I found that the version `0.7.1` does not have this bug.

**Comment by mbaker3** [comment]
While the error no longer shows in `0.7.1` the extension also doesn't seem to validate plugin paths at all. 

`0.7.2` seems to work as expected and correctly validates the plugin/module path.

**Note:** This issue also affects the `Preview config` and `Preview modifier` commands which don't correctly display the built time changes from the plugins. The plugin changes do work for actual builds.

**Comment by byCedric** [comment]
This also happens for monorepos where the plugin package is installed in the root of the workspace.

**Comment by freeridre** [comment]
Hi,
I have the same problem. [I have written it in expo’s forum](https://forums.expo.dev/t/expo-config-module-not-found/58174)

**Comment by byCedric** [comment]
Fixed by #71, hopefully we can release it this week!

**Comment by byCedric** [comment]
As mentioned in #71 

> 🎉 This PR is included in version 0.7.4 🎉
> 
> The release is available on [GitHub release](https://github.com/expo/vscode-expo/releases/tag/0.7.4)
> 
> Your **[semantic-release](https://github.com/semantic-release/semantic-release)** bot 📦🚀

If there is anything else, please let us know! 😄
